lib/sufia/engine.rb

```diff
@@ -11,6 +11,11 @@ class Engine < ::Rails::Engine
       #{Hydra::Engine.root}/app/models/concerns
     )
 
+    config.eager_load_paths += %W(
```

Comment:
Please add a comment about why this is necessary. e.g. `Force these models to be added to Legato's registry in development mode`